### PR TITLE
add zenodo.json

### PR DESCRIPTION
**Pull request by Remi-Gau**
To make it easy to curate authors on zenodo releases

### CODE REVIEW

**Comment by Remi-Gau** [comment]
fixes #225

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  "creators": [
+    {
+      "affiliation": "Université Catholique de Louvain",
+      "name": "Gau, Rémi",
+      "orcid": "0000-0002-1535-9767"
+    },
+    {
+      "affiliation": "Université Catholique de Louvain",
+      "name": "Barilari, MArco",
+      "orcid": ""
```
Comment (Remi-Gau) [inline .zenodo.json:13]: @marcobarilari 
can you add your ORCID number here?

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    },
+    {
+      "affiliation": "Université Catholique de Louvain",
+      "name": "Battal, Ceren",
+      "orcid": ""
```
Comment (Remi-Gau) [inline .zenodo.json:18]: @CerenB  
can you add your ORCID number here?

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    },
+    {
+      "affiliation": "Université Catholique de Louvain",
+      "name": "Rezk, Mohamed",
+      "orcid": ""
```
Comment (Remi-Gau) [inline .zenodo.json:23]: @mohmdrezk can you add your ORCID number here?

File: .zenodo.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    },
+    {
+      "affiliation": "Université Catholique de Louvain",
+      "name": "Collignon, Olivier",
+      "orcid": ""
+    },
+    {
+      "affiliation": "Université Catholique de Louvain",
+      "name": "Gurtubay, Ane",
+      "orcid": ""
```
Comment (Remi-Gau) [inline .zenodo.json:33]: @anege  
can you add your ORCID number here?

File: .zenodo.json [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    },    
+    {
+      "affiliation": "Université Catholique de Louvain",
+      "name": "Falagiarda, Federica",
+      "orcid": ""
```
Comment (Remi-Gau) [inline .zenodo.json:38]: @fedefalag  
can you add your ORCID number here?

File: .zenodo.json (hunk 2 above)
Comment (CerenB) [inline .zenodo.json:18]:
```suggestion
      "orcid": "0000-0002-9844-7630"
```

**Review by CerenB — CHANGES_REQUESTED** [review]
I've added mine if you do not see it: https://orcid.org/0000-0002-9844-7630

File: .zenodo.json (hunk 1 above)
Comment (marcobarilari) [inline .zenodo.json:13]:
```suggestion
      "orcid": "0000-0002-3313-3120"
```

**Comment by marcobarilari** [comment]
thanks @Remi-Gau

**Review by marcobarilari — APPROVED** [review]

**Pull request merged**